graphdatascience/tests/integration/test_graph_ops.py

```diff
@@ -424,14 +424,23 @@ def test_graph_nodeProperty_stream_raise_error_with_duplicate_keys(gds: GraphDat
         gds.graph.nodeProperty.stream(G, "x", db_node_properties=["x", "z", "name"], concurrency=2)
 
 
-@pytest.mark.enterprise
+def test_graph_nodeProperty_fail_on_duplicate_node_properties(gds: GraphDataScience) -> None:
+    G, _ = gds.graph.project(GRAPH_NAME, {"Node": {"properties": ["x", "y"]}}, "*")
+
+    with pytest.raises(ValueError, match="Duplicate values in db_node_properties."):
+        gds.graph.nodeProperty.stream(G, "x", db_node_properties=["z", "z", "name"], concurrency=2)
+
+
+# @pytest.mark.enterprise
```

Comment:
intended comment here or?

Reply:
if intended, we should delete the line

Reply:
ah it's arrow, so it should not be commented
```suggestion
@pytest.mark.enterprise
```